mandatory/2-function-creation.js

```diff
@@ -4,7 +4,14 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
```

Comment:
Nice 👍 
Can you think of a shorter way to write this? Maybe you don't need the `if` statement? Can you write it with just a `return` statement?